cmd/storj-sim/network.go

```diff
@@ -24,7 +24,9 @@ import (
 	"storj.io/storj/internal/dbutil/pgutil"
 	"storj.io/storj/internal/fpath"
 	"storj.io/storj/internal/processgroup"
+	"storj.io/storj/lib/uplink"
 	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/storj"
```

Comment:
storj-sim was intentionally trying to keep dependencies minimal, so it would be relatively independent of the binary versions, since we need to test multiple versions of things at some point

Reply:
i.e. Use the binaries to achieve what you want to do.

Reply:
This is used to create a serialized scope to pass to the gateway for configuration. Do you have any good suggestions on how to do that? I see no reason for any command to need to do that.

Reply:
Here we should have something that creates a uplink ready for action that has the necessary rights. Currently we've been hacking gateway to do that, even our tests use `storj-sim network env GATEWAY_0_DIR` to find the necessary information.

However the better approach would be to have the capability to add/remove users on the command-line, create scopes etc. this also comes handy for testing. The storj-sim in some terms is also a test of the tooling or how easily we can interact with the API without resorting to code.

Unfortunately we have already cut corners in quite a few places (making graphql requests).

We can leave for now, but yeah, not happy about it.

Reply:
Alright, I'll keep this in mind when I'm building out the `uplink` command's features for creating/viewing/importing scopes.